.github/workflows/r-cmd-check.yml

```diff
@@ -81,11 +82,33 @@ jobs:
           GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       #####################c#####################
 
+      - name: Check Version
+        id: check_version
+        run: |
+          maintenance_version="F"
```

Comment:
Here this is a step to check if version has this pattern x.y.x.m with m >= 9000 - if this is the case then on next step we are doing this : Sys.setenv("_R_CHECK_CRAN_INCOMING_SKIP_LARGE_VERSION_" = TRUE) to avoid having notes on rcmdchecks

Reply:
ah sure? (because I see this more like temp setup to just enable checks in case of dev version) - but np to change this !